benchbuild/project.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=bad-continuation
```

Comment:
Codacy found an issue: [Useless suppression of 'bad-continuation'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)